mitmproxy/io/compat.py

```diff
@@ -300,6 +300,12 @@ def convert_11_12(data):
     return data
 
 
+def convert_12_13(data):
```

Comment:
There is a race condition with your other PR here, we now need to transition from 13 to 14 :)